addon/components/md-nav.js

```diff
@@ -0,0 +1,19 @@
+import Ember from 'ember';
+import layout from '../templates/components/md-nav';
+
+const { run: { next }, Component } = Ember;
+
+export default Component.extend({
+  classNames: ['md-nav'],
+  layout,
+  tagName: 'nav',
```

Comment:
navbar-fixed is supposed to wrap the component. It's not a class that's added to the component's root element.

I would expect this to work
```hbs
<div class="navbar-fixed">
  {{#md-nav}}
  {{/md-nav}}
</div>
```
If it does not, please create a small example that I can take a look at. A simpler way to debug this kind of thing is to identify where the HTML that this library is building is different than what's specified in the [Materialize conventions](http://materializecss.com/navbar.html).